src/Editor.elm

```diff
@@ -160,19 +161,33 @@ update msg ({ present } as model) =
 
         JsonRead contents ->
             case contents |> Helpers.Exporting.Json.Decode.decode of
-                Ok t ->
+                ( Ok cfg, Ok t ) ->
                     ( UndoList.new
-                        { present | jsonImport = None, tableau = t }
+                        { present | jsonImport = None, config = cfg, tableau = t }
                         model
                     , cache contents
                     )
 
-                Err e ->
+                ( Ok cfg, Err t ) ->
+                    ( UndoList.new
+                        { present | jsonImport = ImportErr (Json.Decode.errorToString t), config = cfg }
+                        model
+                    , cache contents
+                    )
```

Comment:
I think importing config without a tableau is rather pointless, i.e., don’t change the config here, just show the tableau decoding error.